wiki/Tournaments/TWC/2015/de.md

```diff
@@ -2,155 +2,526 @@
+   - Um für die Liste akzeptiert zu werden, musst du sicherstellen dass dein globaler Taiko-Performance-Rang über \#8000 liegt.
+   - Um für die Liste akzeptiert zu werden, musst du sicherstellen dass du die [osu!-Communityregeln](/wiki/Rules) in den letzten 12 Monaten nicht gebrochen hast.
+3. Um dich anzumelden, [sendee eine PM an Loctav](https://osu.ppy.sh/home/messages/users/71366) mit dem Titel “TWC Registration”.
+   - Füge deine lokale Zeitzone zum Registrierungs-PM hinzu.
+   - Du erhältst eine Bestätigung, dass deine Anmeldung erfolgreich angekommen ist. Deine Anmeldung ist dann in Bearbeitung.
 4. Die maximale Anzahl an Teams liegt bei 32.
-   1. Die Gesamtanzahl an Teams hängt von den eingegangen Anmeldungen ab.
+   - Die Gesamtanzahl an Teams hängt von den eingegangen Anmeldungen ab.
 5. Alle erfolgreich zusammengestellten Teams werden nach der Registrierungsphase veröffentlicht.
-6. Map Wähler dürfen nicht am Turnier als Spieler teilnehmen.
+6. Beatmap-Wähler dürfen am Turnier nicht als Spieler teilnehmen.
 
 ### Etappenanweisungen
 
-1. In der ersten Etappe (Gruppenphase) werden alle Teams in 8 Gruppen mit jeweils 4 Teams aufgeteilt.
-   1. Dies hängt jedoch an der Gesamtanzahl an Teams ab und kann daher variieren.
+1. In der ersten Etappe (Gruppenetappe) werden alle Teams in 4 Gruppen mit jeweils 6 Teams aufgeteilt.
+
 2. Die Teams werden gegen alle anderen Teams in derselben Gruppe antreten.
-3. Rankings werden für jede Gruppe durch eine Sortierung von den Ergebnissen anhand der Performance jedes Teams an den folgenden Prioritäten ermittelt:
-   1. Am meisten gewonnene Partien.
-   2. Hat ein größeres `{(Anzahl an gewonnenen Maps) - (Anzahl an verlorenen Maps)}`.
-   3. Am meisten gewonnene Maps.
-   4. Hat ein größeres `∑{(gesamtes Ergebnisdifferenz) / (maximales Ergebnis)}`.
-   5. Gewinner von Rückspielen.
-4. Die besten 2 Teams in der Gruppe kommen in die Double-Elimination Etappen.
-   1. Dies hängt jedoch an der Gesamtanzahl an Teams ab und kann daher variieren.
-5. Die folgenden Etappen sind Double-Elimination Etappen. Dies bedeutet, dass der Gewinner normal in die nächste Etappe kommt und das andere Team in die Verliereretappe kommt.
-6. Basierend auf [dieser Grafik](https://puu.sh/bUq5V/f1066103b0.png) teilen sich die Etappen wie folgt auf:
+
+3. Rankings für jede Gruppe werden durch eine Sortierung von den Ergebnissen anhand der Performance jedes Teams anhand der folgenden Prioritäten ermittelt:
+   1. Meiste gewonnene Partien.
+   2. Hat eine größere `{(Anzahl an gewonnenen Maps) | (Anzahl an verlorenen Maps)}`.
+   3. Meiste gewonnene Maps.
+   4. Hat eine größere `∑{(gesamte Ergebnisdifferenz) / (maximales Ergebnis)}`.
+   5. Gewinner des Rückspiels.
+
+4. Die besten 4 Teams in der Gruppe gehen weiter in die Knock-Out-Etappen.
+
+5. Die folgenden Etappen sind Double-Elimination-Etappen. Dies bedeutet, dass der Gewinner in zur nächsten Etappe fortfährt und das andere Verliererteam in die Verliererklasse versetzt wird.
```

Comment:
[One article](https://osu.ppy.sh/wiki/de/Tournaments/OWC/2014) uses `Doppeleleminierungsetappen`. But this is fine too.

Reply:
Somehow missed that, will change for consistency.